Gemfile.lock

```diff
@@ -246,7 +246,7 @@ GEM
     sshkit (1.8.1)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
-    stanford-mods (2.1.0)
+    stanford-mods (2.2.0)
```

Comment:
2.2.0  has the imprint_display_str,  so this Gemfile.lock  is enough for this particular update.